docs/rules/periphery/periphery-split-csv.py

```diff
@@ -75,7 +75,7 @@ def csv_fname(self):
 
 
 data = [[]]
-for l in open('periphery.csv'):
+for l in open('periphery.csv', encoding='utf8'):
```

Comment:
On my machine, I needed to specify utf8 encoding to make the file open properly, or else we would get an exception. 

I can remove this for you if you feel it's unnecessary to specify that here.

Reply:
This seems fine!